test/blockstore-test.js

```diff
@@ -281,12 +281,20 @@ describe('BlockStore', function() {
   });
 
   describe('FileBlockStore (Unit)', function() {
-    const location = '/tmp/.bcoin/blocks';
+    const location = () => {
+      switch (process.platform) {
+        case 'win32':
+          return '\\tmp\\.bcoin\\blocks\\';
```

Comment:
What does this refer to on windows ?

Reply:
As this is a unit test, this path is not meant to be used. It's just a placeholder to make it easier to analyze the test case.